docs/Introduction.Android.md

```diff
@@ -152,6 +152,7 @@ In your app’s `buildscript` (i.e. `android/app/build.gradle`) add this in `dep
 dependencies {
     // ...
     androidTestImplementation('com.wix:detox:+')
+    implementation 'androidx.appcompat:appcompat:1.1.0'
```

Comment:
This is definitely the right page for this but I don't like it under this section (named, "Add the native detox dependency"). 'Need to put things under the right context...